docs/content/observatorium/thanos/thanos_programmatic_access.md

```diff
@@ -12,7 +12,7 @@ You can use your personal token for testing and ad-hoc scripts, but it will expi
 2. Copy the token from the User dropdown menu
 3. Use this token as your oauth bearer token when connecting to Thanos via the client library
    * Eg:
-   ```python
+   ```
```

Comment:
does the "python" formatting not work?

Reply:
jupyterbook build gave out a warning for it: 

```
/home/hukhan/projects/github/operate-first/apps/docs/content/observatorium/thanos/thanos_programmatic_access.md:15: WARNING: Could not lex literal_block as "python". Highlighting skipped.
```